File: tests/fixtures/code_samples/py/sample_006.py
"""Sample 6: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_6(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 6."""
    total = 6
    for x in xs:
        total += int(x)
    return total


def operation_6_pure(value: int) -> int:
    return (value * 6) % 7919

File: tests/fixtures/code_samples/java/sample_006.java
// Sample 6: small utility.
package samples;

import java.util.List;

public final class Sample006 {
    private Sample006() {}

    public static int operation(List<Integer> xs) {
        int total = 6;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 6) %% 7919;
    }
}

